config/application.rb

```diff
@@ -175,6 +175,22 @@ class Application < Rails::Application
     # like if you have constraints or database-specific column types
     config.active_record.schema_format = :sql
 
+    # Starting 6.1.6.1 Rails forced users to explicitly list all the classes
+    # that are allowed to be loaded by Psych
+    # (https://github.com/rails/rails/pull/45584#issuecomment-1183255990)
+    # But this also makes impossible to test Rails 6.1.6.1+ on Ruby 2.5
+    # because in 2.5 safe_load doesn't have keywords that are used by Rails
```

Comment:
Noting what we mentioned on IRC: Foreman 3.4 (current develop) will drop Ruby 2.5. Foreman 3.4 is also the only one on Rails 6.1 so rather than hacking around it we should pull the trigger and drop Ruby 2.5.

I've been told @ehelms was looking into that.

Reply:
But just in case we still want to support it: https://github.com/rails/rails/issues/45590.

Reply:
PR to drop this from our jobs -- https://github.com/theforeman/jenkins-jobs/pull/242